pkgs/build-support/docker/default.nix

```
@@ -56,6 +57,15 @@ let
     done;
   '';
 
+  # Map nixpkgs architecture to Docker notation
```

```suggestion
  # Map nixpkgs architecture to Docker notation
  # Reference: https://docs.docker.com/docker-for-mac/multi-arch/
```

Unless you have a more specific link.